### ISSUE TITLE
Support websocket 5.0.0

### ISSUE THREAD

**Issue by Ashark** [leafac/obs-cli#5]
Hi.
OBS updated to v 28, and it switched to web socket 5.0.0.
The compatibility is made with 4.9.1-compat socket.
Do you want to update the project to support new version of obs-websocket?

**Comment by leafac** [comment]
Yes, I’m planning on doing that, but I haven’t gotten around to it yet…

**Comment by heathdutton** [comment]
Eager for this

**Comment by pschmitt** [comment]
In the meantime you can give this a try (yes, it supports the new API):

https://github.com/pschmitt/obs-cli
